blueflood-integration-tests/src/integration-test/java/com/rackspacecloud/blueflood/io/ElasticIOIntegrationTest.java

```diff
@@ -67,7 +67,7 @@ public static void startElasticsearch() {
 
     @Before
     public void setup() throws Exception {
-        helper = ElasticsearchRestHelper.getConfigurableInstance();
+        helper = ElasticsearchRestHelper.getInstance();
```

Comment:
You ran this test with the singleton change? I thought I remembered tests that seemed like they'd fail if this always returned the same object.

Reply:
Yes. It ran successful. just to make sure I ran it multiple times. It did not complain.
```
Tests run: 39, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 54.647 sec - in com.rackspacecloud.blueflood.io.ElasticIOIntegrationTest

Results :

Tests run: 39, Failures: 0, Errors: 0, Skipped: 0
```